src/masternodes/balances.h

```diff
@@ -101,12 +103,12 @@ struct CBalances {
         return str;
     }
 
-    static CBalances Sum(const std::vector<CTokenAmount> &tokens) {
+    static ResVal<CBalances> Sum(const std::vector<CTokenAmount> &tokens) {
         CBalances res;
         for (const auto &token : tokens) {
-            res.Add(token);
+            Require(res.Add(token));
```

Comment:
Same as above. Let's aim to add a TODO / WARNING note above, and not break early to remain compatible.

Reply:
8c0d1c6dd